pdr_backend/pdr_dashboard/util/db.py

```diff
@@ -25,14 +25,15 @@
 )
 from pdr_backend.util.constants_opf_addrs import get_opf_addresses
 from pdr_backend.util.time_types import UnixTimeMs
+from pdr_backend.lake.lake_mapper import LakeMapper
 
 logger = logging.getLogger("predictoor_dashboard_utils")
 
 
```

Comment:
There is no code for handling parquet exports, just for reading from them

Reply:
If we need to create another DuckDB object inside the dashboard then a new connection is created and we run into concurrency issue, because the object automatically creates a connection inside constructor